numba_dppy/compiler.py

```diff
@@ -196,6 +197,8 @@ def compile_kernel_parfor(sycl_queue, func_ir, args, args_with_addrspaces, debug
 
 
 def compile_dppy_func(pyfunc, return_type, args, debug=False):
+    if DEBUG:
+        debug = True
```

Comment:
That doesn't feels right. Why we are not passing correct `debug` flag instead?

Reply:
Let's remove DEBUG from compiler.py and move to the place, where compile_dppy_func is called?